temporalio/worker/workflow_sandbox/_restrictions.py

```diff
@@ -99,6 +99,17 @@ class methods (including __init__, etc). The check compares the against the
     fully qualified path to the item.
     """
 
+    passthrough_all_modules: bool = False
+    """
+    Pass through all modules, do not sandbox any modules. This is the equivalent
+    of setting :py:attr:`passthrough_modules` as every module ever imported into
```

Comment:
"as" doesn't seem like the right word here.

Reply:
I understand what it's saying now. Yes, "to" would be better; how about this:

> This is the equivalent of setting :py:attr:`passthrough_modules` to a list of all modules imported by the workflow.

Reply:
Fixed